docs/docs/latest/topics/using/edgectl/edge-control.md

```diff
@@ -26,327 +27,161 @@ chmod a+x edgectl
 mv edgectl ~/bin  # Somewhere in your PATH
 ```
 
-Note: Similar instructions work for Windows:
+### Upgrading
+
+Make sure you've terminated the daemon.
 
 ```bash
-curl -fLO https://metriton.datawire.io/downloads/windows/edgectl.exe
-mv edgectl.exe C:\windows\  # Somewhere in your PATH
+$ edgectl quit
+Edge Control Daemon quitting...
 ```
 
-but Edge Control’s cluster features, as described in this document, do not work correctly on Windows at this time.
+Download the latest binary, as above, and replace your existing binary.
 
-Note: You can build Edge Control from source, but the straightforward way
+## Service Preview Quick Start
 
-```bash
-GO111MODULE=on go get github.com/datawire/ambassador/cmd/edgectl`
-```
+Service Preview creates a connection between your local environment and the cluster. These connections are managed through the Traffic Manager, which is deployed in your cluster, and the `edgectl` daemon, which runs in your local environment.
 
-leaves you with a binary that has no embedded version number. If you really want to build from source, clone the repository and run `./builder/build_push_cli.sh build`, which will leave a binary in the `~/bin directory`. We will have a better answer for building from source soon.
+There are three basic commands that are used for Service Preview:
 
-2. Launch the daemon component using `sudo`:
+1. Launch the edgectl daemon:
 
 ```bash
 $ sudo edgectl daemon
-Launching Edge Control Daemon v1.0.0-ea5 (api v1)
+Launching Edge Control Daemon v1.3.2 (api v1)
 ```
 
-In order to mediate traffic to your clusters, Edge Control inserts itself into the DNS for your host (this is why it requires root access to run). It intercepts queries to your system’s primary DNS server, responds to queries that have to do with connected clusters, and forwards any other queries on to a fallback DNS server.
-
-By default, the daemon intercepts queries to the primary DNS server listed in `/etc/resolv.conf`, and uses Google DNS on 8.8.8.8 or 8.8.4.4 for its fallback DNS server. You can override the choice of which DNS server to intercept using the `--dns` option, and you can override the fallback server using the `--fallback` option. For example, if `/etc/resolv.conf` is correct, but you have a local DNS server available on 10.0.0.1 that should be used for non-cluster queries, you could run
+2. Connect your laptop to the cluster. This will enable your local environment to initiate traffic to the cluster.
 
-```bash
-$ sudo edgectl daemon --fallback 10.0.0.1
-Launching Edge Control Daemon v1.0.0-ea5 (api v1)
+```
 ```
```

Comment:
Did you mean to add a newer example here?

```console
$ edgectl connect
Connecting to traffic manager in namespace ambassador...
Connected to context k3s-default (https://172.20.0.3:6443)
```

Reply:
Oh, perhaps you're dropping the examples in this section, since there's nothing for setting up an intercept either. In that case you need to drop the empty code block and maybe the example above for launching the daemon.